src/sentry/static/sentry/app/views/eventsV2/landing.tsx

```diff
@@ -274,6 +292,14 @@ class DiscoverLanding extends AsyncComponent<Props, State> {
     );
   }
 
+  togglePrebuilt = () => {
+    const {renderPrebuilt} = this.state;
+
+    setRenderPrebuilt(!renderPrebuilt);
```

Comment:
Maybe this should be in the callback of `setState` once react finishes updating the state?